crates/conjure_core/src/rules/cnf.rs

```diff
@@ -41,25 +41,28 @@ fn remove_implication(expr: &Expr, _: &SymbolTable) -> ApplicationResult {
 /// Converts an equivalence to cnf
 ///
 /// ```text
-/// x <-> y ~>  (x /\ y) \/ (!x /\ !y)
+/// x = y ~>  (!x \/ y) /\ (x /\ !y)
```

Comment:
good idea! the comment has a typo in it now though.

to better explain why this is correct, maybe change the comment o:

`x = y ~~> x -> y /\ y -> x ~~> (!x \/ y) /\ (!y \/ x)`

is this clearer to you?

feel free to add more explanation / better type-setting etc, if possible.